modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/helper/EpisodesStartedAndCompleted.kt

```diff
@@ -0,0 +1,12 @@
+package au.com.shiftyjelly.pocketcasts.models.db.helper
+
+data class EpisodesStartedAndCompleted(
+    val started: Int,
+    val completed: Int,
+) {
+    val percentage: Double = if (started == 0) {
+        100.0
```

Comment:
I kind of wonder if `0.0` might make more sense here since if they haven't started any episodes they also wouldn't have finished any episodes. This is a pretty extreme edge case though, so I don't feel remotely strongly.